lib/api/src/externals/function.rs

```diff
@@ -22,21 +23,22 @@ use wasmer_vm::{
 };
 
 /// A function defined in the Wasm module
-#[derive(Clone, PartialEq)]
+#[derive(Clone, PartialEq, MemoryUsage)]
 pub struct WasmFunctionDefinition {
     // Address of the trampoline to do the call.
+    #[memoryusage(ignore)]
```

Comment:
Consider `#[loupe(ignore)]` or `#[loupe(skip)]`. I think it's more common to use the crate name as a the unique first element in the attribute than the name of the trait. For example serde, structopt, `WasmerEnv`.

Serde even does `#[serde(skip_serializing)]` and `#[serde(skip_deserializing)]` to disambiguate the two

Reply:
I agree. Let's move to `#[loupe(skip_memoryusage)]`.